xml/hardening_pam_stack.xml

```diff
@@ -255,16 +255,16 @@ session  include        common-session
 #session  optional       pam_lastlog.so nowtmp showfailed
 session  optional       pam_mail.so standard</screen>        
      <para>
-      You may also lock out root, though obviously you must be very certain
+      You can also lock out root, though obviously you must be very certain
```

Comment:
inconsistent spelling? "lock out" here but in line 263 it is "lockout". according to MS terminology the more common writing is "lockout" and "lock out" delivers no results

Reply:
changed